client/ui/controllers/settingsController.cpp

```diff
@@ -144,7 +146,12 @@ void SettingsController::clearSettings()
     m_serversModel->resetModel();
     m_languageModel->changeLanguage(
             static_cast<LanguageSettings::AvailableLanguageEnum>(m_languageModel->getCurrentLanguageIndex()));
-    m_sitesModel->setRouteMode(Settings::RouteMode::VpnAllSites);
+
+    m_sitesModel->setRouteMode(Settings::RouteMode::VpnOnlyForwardSites);
```

Comment:
Why we use VpnOnlyForwardSites by default with Split Tunneling false? 
VpnAllSites looks better in this case.

Reply:
please check https://github.com/amnezia-vpn/amnezia-client/pull/744/files#diff-2a5303f3e078360a1dbb1ff0eda7b8d3da2268a2e39aee2f08de98c99206379dR76